tutorials/klusterai-api/rag.ipynb

```diff
@@ -0,0 +1,530 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "566e7e41",
+   "metadata": {},
+   "source": [
+    "# RAG + Embeddings Endpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8566fe2a",
+   "metadata": {},
+   "source": [
+    "We'll learn how to use our [embeddings endpoint](https://docs.kluster.ai/api-reference/reference/#create-embeddings) in a Retrieval Augmented Generation (RAG) pipeline with PDF document support using [LlamaIndex](https://docs.llamaindex.ai/en/stable/module_guides/models/embeddings/)\n",
+    "\n",
+    "\n",
+    "**Models:**\n",
+    "1. **Embeddings**: Leveraging the [BAAI/bge-m3](https://platform.kluster.ai/models) model.\n",
+    "2. **Language Model (LLM) for Querying**: Utilizing the [meta-llama/Llama-4-Maverick-17B-128E-Instruct-FP8](https://platform.kluster.ai/playground?model=meta-llama/Llama-4-Maverick-17B-128E-Instruct-FP8) model.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82108a06",
+   "metadata": {},
+   "source": [
+    "### Setup and Pre-requisites\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6e42cda8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install the necessary packages, including the PDF reader for LlamaIndex\n",
+    "%pip install llama-index llama-index-llms-openai-like llama-index-embeddings-openai-like llama-index-readers-file requests"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25590afc",
+   "metadata": {},
+   "source": [
+    "### API Keys and Model Endpoints\n",
```

Comment:
```suggestion
    "### API keys and model endpoints\n",
```

Reply:
Kluster prefers no capitalization after the first one, please change the other headings too :)